app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/di/MovieDetailsModule.kt

```diff
@@ -0,0 +1,18 @@
+package agency.five.codebase.android.movieapp.ui.moviedetails.di
+
+import agency.five.codebase.android.movieapp.ui.moviedetails.MovieDetailsViewModel
+import agency.five.codebase.android.movieapp.ui.moviedetails.mapper.MovieDetailsMapper
+import agency.five.codebase.android.movieapp.ui.moviedetails.mapper.MovieDetailsMapperImpl
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val movieDetailsModule = module {
+    viewModel {
```

Comment:
Instead of setting movieId to `0` and then changing it, you can pass id here as parameter
```
viewModel { (movieId: Int) ->
   MovieDetailsViewModel(
      movieId = movieId,
      ....
```